#2072 home page update

twjeffery opened the pull request:
Issue: https://github.com/GovAlta/ui-components/issues/2072


[Figma Design here](https://www.figma.com/design/IeqrkkSdOjbBtWHLM02Wv1/Update-to-DS-website-pages?node-id=504-179137&t=gXvhxwXC8cyaHU3V-4)

![image](https://github.com/user-attachments/assets/33b969de-16e4-4c83-97e2-cc40fe3f4b64)

bdfranck commented:
- ✅ I can see the updated "Using the design system" section
- ✅ I can see the new "What happens when I need a new thing" section
- ✅ I can see the sections stack on mobile

<img width="1311" alt="image" src="https://github.com/user-attachments/assets/d4341d42-b532-4d6a-8a5b-a4e1e0e513e3" />

<img width="594" alt="image" src="https://github.com/user-attachments/assets/8d1b07da-baa0-408d-959b-350493b747e3" />

bdfranck commented:
- ✅ When I select "Go to UX practice resources", I can see the DDD UX Sharepoint.
- ✅ When I select "Common capabilities", I can see the Common capabilities site.

bdfranck reviewed (approved):
Looks good, @twjeffery! Will the below list from the [mockups](https://www.figma.com/design/qWGWDvULAAZuVaDpXCRzqP/DS-Reference-website?node-id=14038-152836&t=G8auZON7y4FoejAW-1) be implemented in a future PR?

<img width="618" alt="image" src="https://github.com/user-attachments/assets/66b10ec2-1e37-499b-aa1d-2de27372f9c9" />

twjeffery commented:
> Looks good, @twjeffery! Will the below list from the [mockups](https://www.figma.com/design/qWGWDvULAAZuVaDpXCRzqP/DS-Reference-website?node-id=14038-152836&t=G8auZON7y4FoejAW-1) be implemented in a future PR?
> 
> <img alt="image" width="618" src="https://private-user-images.githubusercontent.com/1479091/440943285-66b10ec2-1e37-499b-aa1d-2de27372f9c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY2NTIzMjAsIm5iZiI6MTc0NjY1MjAyMCwicGF0aCI6Ii8xNDc5MDkxLzQ0MDk0MzI4NS02NmIxMGVjMi0xZTM3LTQ5OWItYWExZC0yZGUyNzM3MmY5YzkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDUwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA1MDdUMjEwNzAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTI0YzJjNWRiYjhlYjhiMmViY2ViZGYzZmFlNjQ5NmM2N2U1MDMyMmY1YThkMTQ4ZjM4MzA2YmIxOTc2Mjc3ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.lVPvEkdHmEPrsa08NMauOL2AC34GaFbIdZzFFc0s5iI">

Great catch @bdfranck, that content got lost in the updated design.

I've added that additional content to the [Get started with the design system](https://deploy-preview-360--abgov-ui-component-docs.netlify.app/get-started) page. That way the home page is kept more high level, and we can dig into the details on the _learn more_ link.

[Updated design for get started with the design system page in Figma](https://www.figma.com/design/IeqrkkSdOjbBtWHLM02Wv1/Updates-to-DS-website-pages?node-id=556-78630&t=eBM7jtHgmxurzpUz-4)

![image](https://github.com/user-attachments/assets/9a610d84-dc53-4c20-b75e-50669f5acd8a)


## Other changes after another review
- I updated the styling for the card-lite component to not have to use any `!important` classes

bdfranck reviewed (approved):
I looked at the preview and I see the "getting started" numbered list on the "Starting with the design system" page. Looks good to me! 👍
